src/Microsoft.Health.ControlPlane.Core/Features/Rbac/RbacService.cs

```diff
@@ -0,0 +1,34 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Health.ControlPlane.Core.Features.Persistence;
+
+namespace Microsoft.Health.ControlPlane.Core.Features.Rbac
+{
+    public class RbacService : IRbacService
+    {
+        private IControlPlaneDataStore _controlPlaneDataStore;
+
+        public RbacService(IControlPlaneDataStore controlPlaneDataStore)
+        {
+            EnsureArg.IsNotNull(controlPlaneDataStore, nameof(controlPlaneDataStore));
+
+            _controlPlaneDataStore = controlPlaneDataStore;
+        }
+
+        public async Task<IdentityProvider> GetIdentityProviderAsync(string name, CancellationToken cancellationToken)
```

Comment:
Is the idea to have additional validation and/or etc here in the future and if so is that implemented in a different PR?

Reply:
Yes, the idea is to expand the RBAC service to have validation and any other non datastore RBAC related code in a single place. In this PR, it's mainly used as an example of how different collections can co-exist.